server/server.go

```diff
@@ -36,6 +36,7 @@ import (
 	"time"
 
 	"context"
```

Comment:
Might as well move context up with the standard go packages if we're reformatting this section.

Reply:
Note that blindly sending all may include, e.g., `Content-Endoding` or other semantic headers.

Reply:
Fixed the context import ordering.

Thanks for catching that.